diff --git a/Configuration.py b/Configuration.py
index 29be069..25de4c3 100644
--- a/Configuration.py
+++ b/Configuration.py
@@ -28,6 +28,11 @@ class GeneralConf():
         self.pieces_joueurN = list()
         self.avantage = None
 
+
+ ######################################################
+ #############SAUVEGARDE/CHARGEMENT PARTIE#############
+ ######################################################       
+
     def sauvegarde_partie(self, joueur):
         """
         @RG
@@ -85,6 +90,10 @@ class GeneralConf():
 
             self.joueur_sauvegarde = int(elements[len(elements) - 1])
 
+ ######################################################
+ ######################################################
+ ######################################################  
+
     def add_piece(self, piece):
         """
         @RG
@@ -138,6 +147,14 @@ class GeneralConf():
             else:
                 self.pieces_joueurN.append(piece)  # NR il a oublié les self
 
+
+    def chemin_parcouru(self, piece):
+        """
+        @RG
+        """
+        #Sauf Roi Cavalier
+        print(piece.PossibleMoves(), piece.nom)
+
     def matrice_affichage(self):
         """
 		@RG
@@ -175,6 +192,13 @@ class GeneralConf():
 		:param pos_arrivee: Destination voulue par le joueur pour la pièce
 		:return bool: Renvoie vrai si le déplacement est autorisé, faux sinon
 		"""
+
+        ##############
+        for piece in self.pieces:
+            if piece.__class__ is Cavalier:
+                self.chemin_parcouru(piece)
+        #############
+
         possible_moves = moves[0]
         possible_eat = moves[1]
 
@@ -228,6 +252,7 @@ class GeneralConf():
             else:
                 self.add_msg_error("Déplacement interdit")
 
+
     def deplacement_piece(self, pos_depart, pos_arrivee, upper):
         """
 		@RG
diff --git a/Main.py b/Main.py
index a4ccf5b..abb4c87 100644
--- a/Main.py
+++ b/Main.py
@@ -107,9 +107,12 @@ def decision_joueur(decision, configuration):
     return [pos_depart, pos_arrivee]
 
 
+
+
 def game_pvp():
     """
     @RG @NR
+    Fonction de jeu pour une partie joueur contre joueur
     """
     configuration = GeneralConf()
 
diff --git a/README.md b/README.md
index e989fe2..81f329b 100644
--- a/README.md
+++ b/README.md
@@ -16,8 +16,8 @@ Python 3.x
 
 ## Lancement du jeu
 Pour lancer le jeu, placez-vous dans le répertoire Polychess et lancez Main.py.
-Tour à tour les joueurs communiquent leurs intentions de jeu selon le format suivant : x1,y1 x2,y2
-x1,y1 représentant les coordonnées initiales de la pièce à déplacer et x2,y2 les coordonnées de la destination de la pièce.
+Tour à tour les joueurs communiquent leurs intentions de jeu selon le format suivant : x1y1 x2y2
+x1y1 représentant les coordonnées initiales de la pièce à déplacer et x2y2 les coordonnées de la destination de la pièce.
 
 ## Trello
 https://trello.com/b/GnWD5gmq/polychess
diff --git a/__pycache__/Configuration.cpython-37.pyc b/__pycache__/Configuration.cpython-37.pyc
index 9eaaf75..04e351c 100644
Binary files a/__pycache__/Configuration.cpython-37.pyc and b/__pycache__/Configuration.cpython-37.pyc differ
